CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 CHANGELOG
 =========
 
+# 4.3.0 / 2020-07-06
+
+* [FEATURE] Record installation information for telemetry and troubleshooting purposes. See [#281][].
+* [BUGFIX] Fix error when facts value doesn't exist on Redhat OS family of the arm architecture. See [#283][]. Thanks to [@kanga333][].
+* [BUGFIX] Fix idempotence when reinstalling same pinned version See [#269][].
+* [DOCS] Update wording on blocking install of problematic agent versions. See [#284][].
```

Comment:
```suggestion
```
Docs are not something we "release", so we can remove it from the changelog.

Reply:
And in general, we can skip changes that are very small/not user-facing.

Reply:
There are some docs changes documented before, but I guess they were bigger/more relevant.